src/main/webapp/app/course/learning-paths/components/learning-path-exercise/learning-path-exercise.component.html

```diff
@@ -0,0 +1 @@
+<ng-content />
```

Comment:
A few notes/questions about your components in general:
1. Please remove the empty scss files 
2. Why are the components standalone? AFAIK we normally have them as part of a module.

For this component:
Is there a specific reason why we wrap the exericse in this component? And why don't we just add it as a child in the HTML

Reply:
1. good point I removed the scss files
2. I used standalone components as their use is strongly recommended by Angular. In general they improve flexibility and modularity. Plus they are and will stay backwards compatible with modules
3. I added a comment in the .ts component file. In this case I can first add the component after setting the learning path mode of the exercise to true. This way the exercise does not try to read the URL for the exercise ID. In the learning path feature I want to prevent reading the ID from the URL as I don't need/and want stable (deep) links here.